packages/projection-typeorm/src/entity/PoolRegistration.entity.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable brace-style */
+import { BigIntColumnOptions, DeleteCascadeRelationOptions } from './util';
+import { BlockEntity } from './Block.entity';
+import { Cardano } from '@cardano-sdk/core';
+import { Column, Entity, JoinColumn, ManyToOne, PrimaryColumn } from 'typeorm';
+import { StakePoolEntity } from './StakePool.entity';
+
+@Entity()
+export class PoolRegistrationEntity {
+  /**
+   * Computed from certificate pointer.
+   * Can be used to sort pool updates.
+   */
+  @PrimaryColumn(BigIntColumnOptions)
+  id?: bigint;
+  @Column()
+  rewardAccount?: Cardano.RewardAccount;
+  @Column(BigIntColumnOptions)
+  pledge?: bigint;
+  @Column(BigIntColumnOptions)
+  cost?: bigint;
+  // Review: should we store this as 'double' instead?
+  // Maybe both formats? If we'll need to do computations with this
```

Comment:
Both might be the way to go, to satisfy both use-cases

Reply:
67d8ebd4f5b92e46c65d1d7874bf989c9de8a4bd